js/popup-window.js

```diff
@@ -0,0 +1,156 @@
+    imagedsk: './img/desktop/Snapshoot1-desktop.svg',
+    spans: ['Canopy', 'Back End Dev', '2015'],
+    textcontent: 'A daily selection of privately personalized reads; no accounts or sign-ups',
+    technologies: ['HTML', 'CSS', 'JAVASCRIPT'],
+    btntxt: 'See Project',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    img: './img/Snapshoot-Portfolio-2.png',
+    imagedsk: './img/desktop/Snapshoot2-desktop.svg',
+    spans: ['Canopy', 'Back End Dev', '2015'],
+    spansdsk: ['Facebook', 'Back End Dev', '2015'],
+    textcontent: 'A daily selection of privately personalized reads; no accounts or sign-ups',
+    technologies: ['HTML', 'CSS', 'javascript'],
+    technologiesdsk: ['HTML', 'CSS', 'Ruby on Rails', 'JavaScript'],
+    btntxt: 'See Project',
+  },
+
+  {
+    title: 'Tonic',
+    titledsk: 'Facebook 360',
+    img: './img/Snapshoot-Portfolio-3.png',
+    imagedsk: './img/desktop/Snapshoot3-desktop.svg',
+    spans: ['Canopy', 'Back End Dev', '2015'],
+    spansdsk: ['Facebook', 'Back End Dev', '2015'],
+    textcontent: 'A daily selection of privately personalized reads; no accounts or sign-ups',
+    technologies: ['HTML', 'CSS', 'javascript'],
+    technologiesdsk: ['HTML', 'CSS', 'Ruby on Rails', 'JavaScript'],
+    btntxt: 'See Project',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    titledsk: 'Uber Navigation',
+    img: './img/Snapshoot-Portfolio-4.png',
+    imagedsk: './img/desktop/Snapshoot4-desktop.svg',
+    spans: ['Canopy', 'Back End Dev', '2015'],
+    spansdsk: ['Uber', 'Back End Dev', '2015'],
+    textcontent: 'A daily selection of privately personalized reads; no accounts or sign-ups',
+    technologies: ['HTML', 'CSS', 'javascript'],
+    technologiesdsk: ['HTML', 'CSS', 'Ruby on Rails', 'JavaScript'],
+    btntxt: 'See Project',
+  },
+
+];
+
+const cardWorks = document.querySelector('#portfolio');
+
+cardWorks.className = 'works';
+for (let i = 0; i < cardetails.length; i += 1) {
+  const workCard = document.createElement('article');
+  const workCardDiv = document.createElement('div');
+  workCardDiv.className = 'card-works';
+  cardWorks.appendChild(workCard);
+  workCard.className = 'card-works';
+  workCard.innerHTML = `<img src= ${cardetails[i].img} alt="snapshootimg" class="snapshoot-img">
+  <img src= ${cardetails[i].imagedsk} alt="snapshootdsk" class="snapshoot-img-dsk">
+  <div class="adjust">
+    <div class="leftblock">
+      <h2 class="card-works-title popup-title">${cardetails[i].title}</h2>
+      <div class="ref">
+        <span class="client">${cardetails[i].spans[0]}</span>
+        <span><i class="fa fa-circle fa-xs"></i></span>
+        <span>${cardetails[i].spans[1]}</span>
+        <span><i class="fa fa-circle"></i></span>
+        <span class="year">${cardetails[i].spans[2]}</span>
+      </div>
+      <p class="card-primary-text">${cardetails[i].textcontent}</p>
+    </div>
+    <ul class="tags">
+      <li class="tag">${cardetails[i].technologies[0]}</li>
```

Comment:
[OPTIONAL] This is an optional but recommended requirement kindly don't skip it otherwise your UI is not looking professional please stick to the Figma design your cards are not the same as the Figma design kindly fix it. Check below :point_down: the comparison your cards and Figma cards :+1: 

**Your Design**  
 ![screencapture-127-0-0-1-5501-2023-04-19-01_32_38 (1)](https://user-images.githubusercontent.com/45226848/232899393-754ad4a8-78ce-44be-be4b-8c8056997a21.png) 

** Figma Design **
 ![Screenshot from 2023-04-19 01-40-29](https://user-images.githubusercontent.com/45226848/232899927-a8efc8a6-23a0-495b-a6d3-27f2df940182.png) |